src/views/portals/YetiView.vue

```diff
@@ -202,66 +207,24 @@ export default {
         },
         zoom: {
           simple: this.$gettext('Area too large'),
-          full: this.$gettext('Current map view is too large. Please zoom to level ' + VALID_FORM_DATA.minZoom),
+          full: this.$gettext('Current map view is too large. Please zoom to level ' + this.validMinimumMapZoom),
```

Comment:
Unfortunately, our translation tools supports only raw string. Well, it's not new from this PR, but we'll have to fix it one day.

Reply:
Nice catch. Fixed!